packages/connect-explorer/src-webextension/background/serviceWorker.ts

```diff
@@ -1,3 +1,12 @@
 /// <reference lib="webworker" />
 
-importScripts('vendor/trezor-connect-webextension.js');
+// in Trezor monorepo we need to specify the path since main points to `src/index.ts`
+// in a published NPM package it points to `build/trezor-connect-webextension.js`
```

Comment:
Wouldn't make sense here to point main to `build/trezor-connect-webextension.js` ? 
Since this package we do not use in any other way?